src/mower_logic/cfg/MowerLogic.cfg

```diff
@@ -11,6 +11,8 @@ gen.add("docking_approach_distance", double_t, 0, "Distance to approach docking
 gen.add("docking_retry_count", int_t, 0, "How often should we retry docking", 4, 1, 10)
 gen.add("outline_count", int_t, 0, "Outline count to mow before filling", 3, 0, 10)
 gen.add("outline_offset", double_t, 0, "Additional outer outline offset, use positive values for safety, negative values to enlarge the area", 0.0, -1.0, 1.0)
+gen.add("mow_angle_offset", double_t, 0, "Mowing angle offset", 0, -180, 180)
+gen.add("mow_angle_offset_is_absolute", bool_t, 0, "If true then the offset is relative to East, if false then the offset is relative to the auto-detected angle.", True)
```

Comment:
Should default to false to preserve existing behavior where mower_configs will not have the env var set

Reply:
Or would it just fail to launch if the env var is not set?